pset-functions.js

```diff
@@ -31,12 +31,27 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+// const playerCardScore = 3
+// const dealerCardScore = 4
+
+function blackJack(playerCardScore, dealerCardScore) {
+  if (playerCardScore <= 21 && dealerCardScore <= 21) return Math.max(playerCardScore, dealerCardScore);
+  if playerCardScorea <= 21) return playerCardScore;
+  if (dealerCardScore <= 21) return dealerCardScore;
+  return = 0
+}
+playerCardScore(19)
+dealerCardScore(21)
 // Add your code above this line
```

Comment:
@ChicoMangusta Good solution here, you got a few tweaks to make in order to make it work.
-  Instead of calling `playerCardScore(19) dealerCardScore(21)` you can call blackJack (19, 22)
- On-Line 39 you're missing a parenthesis and your variable name says `playerCardScorea` but it should say `playerCardScore`.
- On Line 41 when you use the `return` keyword instead of returning 0 like this `return = 0` you would do this `return 0`